xl2times/__main__.py

```diff
@@ -485,6 +485,20 @@ def run(args: argparse.Namespace) -> str | None:
 
     model.files.update([Path(path).stem for path in input_files])
 
+    processing_order = ["base", "subres", "trade", "demand", "scen", "syssettings"]
+    for data_module in processing_order:
+        model.data_modules = model.data_modules + sorted(
+            [
+                item
+                for item in {
+                    DataModule.module_name(path)
+                    for path in input_files
+                    if DataModule.module_type(path) == data_module
```

Comment:
Should we raise a warning/error if there are any files in `input_files` that don't match this if condition for any `data_module in processing_order`? I'm wondering if there might be a bug in the future where we add a new item to `DataModule` but forget to add it to `processing_order`, and then we skip processing some input files.

Reply:
Good idea!